src/apps/vpn/ui/screens/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -280,17 +280,21 @@ VPNViewBase {
     function getPlanText(currencyCode, amount) {
         const amountDisplay = (amount || 0) / 100;
         const localizedCurrency = VPNLocalizer.localizeCurrency(amountDisplay, currencyCode);
+        const isUsOrCa = VPN.countryCode === "US" || VPN.countryCode === "CA"
```

Comment:
Can we use `VPNSubscriptionData.planCurrency` as we did for gating the bundle upsell UI? cc @birdsarah sanity checking that FxA communicates this for App Store and Play Store subs?
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/7301a22f69a1de190001275d645a2c941a70044e/src/apps/vpn/ui/screens/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml#L155-L160

Reply:
We can use `VPNSubscriptionData.planCurrency` here, since the user already has a subscription, but we cannot use it on the IAP screen since the user does not have a subscription at that point.

Should we differ our methods based on whether the user has a subscription or not

CC: @jswinarton @bakulf @birdsarah can we confirm that we should be using currency codes (USD, CAD) instead of country codes (US, CA) for the subscription plan value and NOT the IAP screen?

Reply:
No, fxa doesn't communicate currency. The app stores pick the currency for the user based on that user's signed in setting.